velox/expression/CastExpr.cpp

```diff
@@ -689,12 +689,23 @@ void CastExpr::evalSpecialForm(
   auto fromType = inputs_[0]->type();
   auto toType = std::const_pointer_cast<const Type>(type_);
 
+  SelectivityVector remainingRows = rows;
+  context.deselectErrors(remainingRows);
+  // Skip evaluation if all requested rows have errors. Return a NULL vector.
+  if (!remainingRows.hasSelections()) {
+    context.ensureWritable(remainingRows, toType, result);
```

Comment:
`remainingRows` has no selection here. Should it be `context.ensureWritable(rows, toType, result);`? Same for the next line.